packages/shelter-ui/src/tooltip.tsx

```diff
@@ -87,12 +94,53 @@ export function tooltip(el: HTMLElement, props: Accessor<JSX.Element | [boolean,
     }, 100);
   };
 
-  el.addEventListener("mouseenter", enterHandler);
-  el.addEventListener("mouseleave", exitHandler);
+  const [isInside, setIsInside] = createSignal(false);
```

Comment:
Question: why is this a signal? doesn't look to be doing anything reactive to me

Reply:
Yeah I was questioning same thing to me too. I can me normal variable.